### ISSUE TITLE
Insufficient color contrast for the Certification Project label in light theme

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#64199]
## Description

We have `#f8577c` text against a white background, which results in a contrast ratio of just 3.94.

<img width="385" height="105" alt="Image" src="https://github.com/user-attachments/assets/9379b9f3-8b2e-482a-a85d-7818c12267c5" />

## Affected page

V9 super block pages. 

Example: https://www.freecodecamp.org/learn/responsive-web-design-v9

## Changes needed

The cert project label is using the `love-color` variable, so its values need to be updated.

Changes:
- Replace this hex code with `var(--red70)`
  - https://github.com/freeCodeCamp/freeCodeCamp/blob/b51290d0093a3b6ec35df025ccef3dd87bec5247/client/src/components/layouts/variables.css#L30
  - Note: This variable resolves to `#a94442`. Ref: https://github.com/freeCodeCamp/ui/blob/8e147ecd3e59699acd92fa1a1f6c56aab77afaab/src/colors.css#L50
- Replace this hex code with `var(--red30)`
  - https://github.com/freeCodeCamp/freeCodeCamp/blob/b51290d0093a3b6ec35df025ccef3dd87bec5247/client/src/components/layouts/variables.css#L29
  - Note: This is just for consistency, the hex value doesn't change. Ref: https://github.com/freeCodeCamp/ui/blob/8e147ecd3e59699acd92fa1a1f6c56aab77afaab/src/colors.css#L48
